docs/development/translations.rst

```diff
@@ -0,0 +1,397 @@
+Translations
+============
+
+Quick Start Guide
+-----------------
+
+SecureDrop is a system that lets people share sensitive information
+with investigative journalists anonymously and securely.
+Learn :ref:`more about SecureDrop <what_is_securedrop>`.
+
+SecureDrop is written in English and translated into multiple
+other languages. Translations are managed using **Weblate**,
+a web platform that enables collaborative translation projects.
+
+.. _get_started_using_weblate:
+
+Get started using Weblate
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can choose to :ref:`register on Weblate <how_to_register_on_weblate_email>`
+with your email address, or by linking a GitHub account.
+
+You can contribute to any language, and Weblate has some conveniences
+to make it easier to work with your preferred languages.
+Learn :ref:`how to choose your preferred languages on Weblate <how_to_manage_your_preferred_languages_weblate>`.
+
+Our Weblate instance only contains one project, SecureDrop, which has
+two translation components: the main SecureDrop web application (labeled
+"SecureDrop"), and the translations for the desktop icons of the admin
+and journalist workstations used by news organizations (labeled "desktop").
+
+.. figure:: ../images/weblate/project.png
+   :align: center
+   :figwidth: 80%
+   :width: 100%
+
+   Components of the SecureDrop project in Weblate. Click on the image to see it full-size.
+
+Once a string is translated, it must be approved by a :ref:`reviewer <weblate_glossary_reviewer>`
+before being accepted into SecureDrop.
+
+Approved strings can only be modified by reviewers.
+However, translators can still suggest modifications and make comments
+if they notice something wrong.
+SecureDrop translations are a collaborative endeavour!
+
+Learn :ref:`how to translate SecureDrop using Weblate <how_to_translate_a_language_weblate>`.
+
+.. _getting_help_translations:
+
+Getting help
```

Comment:
This section could be moved up to the end of the introduction, prefaced with something like, "If you're interested in helping with translation and have questions about anything in this document, here's how to ask for help." 

We could add a link to the [Localization Lab SecureDrop project page](https://wiki.localizationlab.org/index.php/SecureDrop) in addition to the Weblate docs. That project page is going to need to be updated to point to `translations.html` instead of `l10n.html` too.

Reply:
I'll move it up before "Get started using Weblate" and check the edition procedures for the Localization Lab wiki.

Reply:
Sounds good. I didn't mean you should update the project page; that was more of a note for us, to increase the odds that it will happen after this is merged. :slightly_smiling_face:  But if you're interested in contributing to their wiki, instructions for requesting access are on the [main page](https://wiki.localizationlab.org/index.php/Main_Page).

Reply:
> This section could be moved up to the end of the introduction [...]

Done.

Reply:
> But if you're interested in contributing to their wiki, instructions for requesting access are on the main page.

Thanks @rmol! I saw that, and I saw **Erin** has updated the page before. I was thinking of sending an email but haven't yet, so if you've more direct communication channels, don't let me stop you! :slightly_smiling_face: